Cubical/Data/Vec/Properties.agda

```diff
@@ -0,0 +1,19 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Data.Vec.Properties where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Data.Nat
+open import Cubical.Data.Vec.Base
+
+private
+  variable
+    ℓ : Level
+
+
+-- This is really cool!
```

Comment:
@Saizan : didn't you experiment with this a long time ago? It's really neat that PathP simplifies this so much

Reply:
beautiful!

Reply:
@mortberg It is really neat! I don't remember doing a lot with vectors though.